environment.yml

```diff
@@ -8,9 +8,11 @@ dependencies:
   - Pillow
   - ipywidgets>=0.6.0
   - requests
+  - mkl
   - pip:
     - pymatsolver
     - discretize
+    - deepdish
     - SimPEG
-    - em_examples
+    - git+https://github.com/geoscixyz/em_examples.git@tdem_groundedsource
```

Comment:
has this been merged? if so, can we go back to depending on the deployed version?